README.md

```diff
@@ -1,107 +1,91 @@
 
 ### Authenticate and instantiate
 
-Install the Ably JavaScript SDK and the Collaborative Spaces SDK:
+Install the Collaborative Spaces SDK:
 
 ```sh
 npm install ably @ably-labs/spaces
 ```
 
-Import the SDKs and then instantiate the Collaborative Spaces SDK with your Ably API key and pass a [clientId](https://ably.com/docs/realtime/authentication?lang=javascript#identified-clients) (if prototyping, you can use a package like [nanoid](https://www.npmjs.com/package/nanoid) to generate an id):
+Instantiate Spaces with your Ably API key and a [clientId](https://ably.com/docs/realtime/authentication?lang=javascript#identified-clients) (if prototyping, you can use a package like [nanoid](https://www.npmjs.com/package/nanoid) to generate an id):
 
 ```ts
 import Spaces from '@ably-labs/spaces';
 
-const spaces = new Spaces({ key: ABLY_API_KEY, clientId: "id" });
+const spaces = new Spaces({ key: ABLY_API_KEY, clientId: 'id' });
 ```
 
-In the above example, the client can be accessed using `spaces.ably` to use functionality in the Ably JavaScript SDK.
+Spaces will create an Ably realtime client for you. This client will be accessible on the instance via `spaces.ably`. See the [Ably docs](https://ably.com/docs) for details.
 
-Alternatively, if you already have an Ably client, you can instantiate by passing it to the `Spaces` constructor:
+Alternatively, if you already have an Ably realtime client, you can instantiate by passing it to the `Spaces` constructor:
 
 ```ts
 import Spaces from '@ably-labs/spaces';
 import { Realtime } from 'ably';
 
-const client = new Realtime.Promise(options)
+const client = new Realtime.Promise(options);
 const spaces = new Spaces(client);
 ```
 
+### CDN
+
+You can also use Spaces with a CDN like [unpkg](https://www.unpkg.com/):
+
+```html
+<script src="https://cdn.ably.com/lib/ably.min-1.js"></script>
+<script src="https://unpkg.com/@ably-labs/spaces@0.0.10/dist/iife/index.bundle.js"></script>
+```
+
+Note that when you use a CDN, you need to include the Ably SDK as well.
+
+## API
+
 ### Space membership
 
-A space is the virtual collaborative area of an application you want to monitor. A space can be anything from a web page, a sheet within a spreadsheet, an individual slide in a slideshow, or the slideshow itself. Create a space and subscribe to events for when clients enter and leave it. Space membership is used to build avatar stacks and display which members are online within a space.
+A space is the virtual, collaborative area of an application you want to monitor. A space can be anything from a web page, a sheet within a spreadsheet, an individual slide in a slideshow, or the slideshow itself. Create a space and listen to events for when clients enter and leave it.
```

Comment:
`from` is incorrect here without a `to`. We should just replace it with a hyphen.